tests/js/app/modules/filter/testType.js

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018 Endless Mobile, Inc.
+
+const Eknc = imports.gi.EosKnowledgeContent;
+
+const Type = imports.app.modules.filter.type;
+const MockFactory = imports.tests.mockFactory;
+
+describe('Filter.Type', function () {
+    const MODELS = ['text/html', 	'video/jpeg'].map(type =>
+        Eknc.ContentObjectModel.new_from_props({
+            content_type: type,
+        }));
+    let filter;
+
+    describe('normal mode', function () {
+        beforeEach(function () {
+            [filter] = MockFactory.setup_tree({
+                type: Type.Type,
+                properties: {
+                    type: 'video/jpeg',
```

Comment:
No such thing as a `video/jpeg` ;-)

Reply:
It could have existed https://www.ietf.org/rfc/rfc3555.txt xD